Add Github niceties (#26)
CleanOceanAction/coa_flask_app: CONTRIBUTING.md

# How to Contribute

> I'm really glad you're reading this, because we are all about Skills Based Volunteering.

Reviewer: ```suggestion
I'm really glad you're reading this because we are all about Skills Based Volunteering.
```

## Reporting Issues

> - If you think its something you can do right now, then skip over this and guide to the good stuff.

Reviewer: ```suggestion
- If you think it's something you can do right now, then skip over this guide and get to the good stuff.
```

> - Have a bug or a feature you want to implement, please report it.

Reviewer: ```suggestion
- If you have a bug or a feature you want implemented, please use our templates to guide you through the process of reporting the issue.
```

- We have templates to guide you through this process for bugs or feature requests.

## Getting Setup

> Just contact one of us if you need any help getting setup.

Reviewer: ```suggestion
Check out the Makefile for the different libraries and tools that this project uses, and then take a shot at installing these dependencies and setting up the development environment for this project. Feel free to contact one of us if you need any help getting set up.
```

## Formatting / Linting / Testing

We are trying to keep the code at a high quality so other developers can have an easier time.
Please make sure to format, lint, and test the code before moving on. The code coverage doesn't
need to be perfect, but at least the changes you have made should have been tested.

> ## Submitting changes

Reviewer: ```suggestion
## Submitting Changes
```

> Please make sure to format, lint, and test the code before moving on. The code coverage doesn't

Reviewer: ```suggestion
Please make sure to format, lint, and test the code before moving on. The code coverage doesn't need to be perfect, but at least the changes you have made should have been tested.
```

> need to be perfect, but at least the changes you have made should have been tested.

Reviewer: ```suggestion
```
